common/src/models/domain/browserComponentState.ts

```diff
@@ -0,0 +1,4 @@
+export class BrowserComponentState {
```

Comment:
Why do we call this Browser?

Reply:
Browser is the only one using it, I'll move it out of here into the browser project

Reply:
Removed 👍